.github/workflows/pull_request.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       fail-fast: false # Don't cancel all on first failure
       matrix:
         os: ["Large_Linux"]  # , "Large_Windows"]
-        python-version: ["3.9", "3.13"]
+        python-version: ["3.13"]
```

Comment:
This should be "3.10", "3.13" ... the goal is to cover the range of supported Python versions, skipping those in between